__fixtures__/v-next/outputv4/akash/deployment/v1beta2/group.ts

```diff
@@ -177,7 +177,7 @@ export const Group = {
     message.groupId = object.groupId !== undefined && object.groupId !== null ? GroupID.fromPartial(object.groupId) : undefined;
     message.state = object.state ?? 0;
     message.groupSpec = object.groupSpec !== undefined && object.groupSpec !== null ? GroupSpec.fromPartial(object.groupSpec) : undefined;
-    message.createdAt = object.createdAt !== undefined && object.createdAt !== null ? Long.fromValue(object.createdAt) : Long.ZERO;
+    message.createdAt = object.createdAt !== undefined && object.createdAt !== null ? (prop => BigInt(prop.toString!!()))(object.createdAt) : BigInt("0");
```

Comment:
what is the idea behind using an executed anonymous function?

Reply:
Ah... I just thought it as a bigint version of Long.fromValue (without applying toString to the property itself). After thinking this again, i'll change it to a decent way of doing this.

Reply:
Done removing.